### PR TITLE
Fixed Spanish Translation for Menu Items

### PR DESCRIPTION
**Pull request by Dimaslg**
Mainly polishing, minor fixes on expressions and translating some words that were not translated.

Changes in Category ballistics
Changes in Category disposable
Changes in Category dragging
Changes in Category explosives
Changes in Category fcs
Changes in Category flashsuppressors
Changes in Category grenades 
Changes in Category interaction
Changes in Category interaction_menu
Changes in Category kestrel4500
Changes in Category laserpointer
Changes in Category laser_selfdesignate
Changes in Category magazinerepoack
Changes in Category medical
Changes in Category microdagr
Changes in Category missileguidance
Changes in Category nametags
Changes in Category nightvision
Changes in Category optionsmenu
Changes in Category overheating
Changes in Category realisticnames
Changes in Category reload:
Changes in Category vehiclelock:
Changes in Category weaponselect:

### CODE REVIEW

File: addons/flashsuppressors/stringtable.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
             <Czech>Tlumič plamene (6,5 mm)</Czech>
             <French>Cache-flamme (6,5 mm)</French>
             <Russian>Пламегаситель (6,5 мм)</Russian>
-            <Spanish>Supresor (6,5 mm)</Spanish>
+            <Spanish>Bocacha (6,5 mm)</Spanish>
```
Comment (esteldunedain) [inline addons/flashsuppressors/stringtable.xml]: I'd prefer to keep "Bocacha"

**Comment by KoffeinFlummi** [comment]
Thanks for your contribution! Please add yourself to the AUTHORS.txt file with a valid email address before we merge this.

**Comment by Dimaslg** [comment]
Already done KoffeinFlummi.

**Comment by bux578** [comment]
#881

**Comment by KoffeinFlummi** [comment]
Thanks!

**Pull request merged**